### DISCUSSION TITLE
Show Category notes on hover

### DISCUSSION THREAD

**Discussion by Deguello** [actualbudget/releases#79] [Feature ideas]
Self-explanatory. Obviously not a huge deal, but it would be nice.

**Comment by bpaulien** [comment]
Another thing I was thinking about along these lines is to have some slightly different icon for a category that already has a note vs one that doesn't.  Because hovering over the category will show the note icon, but I'd like to be able to tell without clicking or hovering over the note icon if there was actually anything in it.  That way I'd know if I should hover over it or not.  If there's no note, no sense in hovering over it.  And if there is a note, I'll know because of the icon.

**Reply by Deguello** [reply]
On the Mac, categories with notes show a grayed document/note icon. Those without a note show nothing. On mouseover, all of them show a dark icon.

**Reply by MikeBishop** [reply]
Same on the PC.
![image](https://user-images.githubusercontent.com/4273797/142249749-b83e0f5e-f425-49c6-8262-1c91e3c97ecc.png)

**Reply by bpaulien** [reply]
Huh!  I'm on the web...  I could have sworn that I had put in some notes, but I guess I didn't.  I just put a note in one of the categories, and you're right.  There's a grayed out icon, whether you're hovering over the category name or not.  So I guess my other comment can be disregarded.  Time to go put in the notes that I thought I had already...  lol

**Comment by bpaulien** [comment]
Also referenced here on issues tab:
#131
